spikeextractors/extractorlist.py

```diff
@@ -49,6 +51,7 @@
     SHYBRIDRecordingExtractor,
     NIXIORecordingExtractor,
     NeuroscopeRecordingExtractor,
+    ALFSortingExtractor,
```

Comment:
This should go in the sorting extractor list

Reply:
oh yes, done